### DISCUSSION TITLE
Allow setting the character for a new line

### DISCUSSION THREAD

**Discussion by TNAJanssen** [outline/rich-markdown-editor#225] [General]
Hi,

We prefer 2 spaces as a line break instead of the \ because the package to render markdown on the backend.

Is there a way to configure this?

**Comment by tommoor** [comment]
There is no way to configure this, the slash formatting is deeply entwined into the markdown parser. This is where it's converted from markdown -> prosemirror AST:
https://github.com/outline/rich-markdown-editor/blob/develop/src/lib/markdown/breaks.ts

And from Prosemirror AST -> Markdown:
https://github.com/outline/rich-markdown-editor/blob/develop/src/nodes/Paragraph.ts#L38
